client/ayon_traypublisher/plugins/create/create_csv_ingest.py

```diff
@@ -388,6 +391,68 @@ def _resolve_repre_path(
 
         return filepath
 
+    def _validate_parents(self, project_name: str, product_item: ProductItem) -> list:
+        """ Ensure parent exists for provided product_item.folder_path
+
+        Args:
+            project_name (str): The project name.
+            product_item (ProductItem): The product item to inspect.
+
+        Returns:
+            list. The parent list if any
+
+        Raise:
+            ValueError: When provided folder_path parent do not exist.
+        """
+        parent_folder_names = product_item.folder_path.lstrip("/").split("/")
+        # Rename name of folder itself
+        parent_folder_names.pop(-1)
+        if not parent_folder_names:
+            return []
+
+        parent_paths = []
+        parent_path = ""
+        for name in parent_folder_names:
+            path = f"{parent_path}/{name}"
+            parent_paths.append(path)
+            parent_path = path
+
+        folders_by_path = {
+            folder["path"]: folder
+            for folder in ayon_api.get_folders(
+                project_name,
+                folder_paths=parent_paths,
+                fields={"folderType", "path"}
+            )
+        }
+        parent_data = []
+        for path in parent_paths:
+            folder_entity = folders_by_path.get(path)
+            name = path.rsplit("/", 1)[-1]
+            folder_type = None
+
+            # Folder exists, retrieve data from existing.
+            if folder_entity:
+                folder_type = folder_entity["folderType"]
+
+            # Define folder type from settings.
+            else:
+                for folder_setting in self.folder_creation_config["folder_type_regexes"]:
+                    if re.match(folder_setting["regex"], name):
+                        folder_type = folder_setting["folder_type"]
+                        break
```

Comment:
For some reason following Path column data is producing following hierarchy with Folder type decorated in `<>`

- `/shots/mw_110_01_0060`

```
/shots <Shot>
└── mw_110_01_0060 <Shot>
```

I believe that since I am using default settings with 
![image](https://github.com/user-attachments/assets/36c274f1-d214-4cf7-833e-0900c0ba7ecd) The types should be defined as:

```
/shots <Shot>
└── mw_110_01_0060 <Folder>
```